src/main/java/org/folio/search/service/setter/AbstractAllValuesProcessor.java

```diff
@@ -77,7 +77,7 @@ private void collectFieldValuesFromEventBody(String path, MultilangValue ctx,
 
   private void collectFieldValuesFromEventBody(String path, MultilangValue ctx, Object v, Predicate<String> filter) {
     if (v instanceof String string) {
-      ctx.addValue(StringUtils.strip(string),
+      ctx.addValue(StringUtils.abbreviate(StringUtils.strip(string), MAX_FIELD_VALUE_LENGTH),
```

Comment:
I'm not sure that we need to use `abbreviate` method in this case. As a result of the method we will have a string with format "string value...", while three dots don't provide any improvement and will be not used while search (search engine will remove them and will not use them in indexing). I would suggest just cut string to the needed length.